controllers/growth.js

```diff
@@ -3,45 +3,208 @@
 var config = require(__dirname + '/../config/config'),
     mysql = require('anytv-node-mysql'),
     util = require(__dirname + '/../helpers/util'),
-    moment = require('moment');
+    moment = require('moment'),
+    crypto = require('crypto'),
+    memcached = require('memcached');
 
 exports.get_channel = function (req, res, next) {
-    var start = function () {
-        var query = '',
-            from = moment().subtract(29, 'day').format('YYYY-MM-DD'), // default
-            to = moment().format('YYYY-MM-DD'), // default
-            data = util.get_data(['id'], [], req.params);
+    var data = {},
+        query_string = {},
+        from = moment().subtract(29, 'day').format('YYYY-MM-DD'), // default
+        to = moment().format('YYYY-MM-DD'); // default
+        data = util.get_data(['id'], [], req.params),
+        query_string = util.get_data([], ['from', 'to'], req.query);
```

Comment:
why not use `Object.assign()`?
`util.get_data(['id'], ['from', 'to'], Object.assign(req.params, req.query))`